data/map.txt

```diff
@@ -27875,7 +27875,7 @@ planet "Ki Patek Ka"
 
 planet "Korati Efreti"
 	attributes efret station
-	landscape land/loc2
+	landscape land/station15
```

Comment:
I'm not too sure on the use of station15 here. The new image looks rather clean, but the old image gave more the impression of a dirty old station that I imagine better fits the reality of the state of Korath stations.

Reply:
Tried changing the image here, does it work?